README.md

```diff
@@ -4,9 +4,16 @@ Team Members: Sukhdeep Kaur, Arash Shamseddini, Tran Doan Khanh Vu, Heidi Ye
 
 Dataset: COVID-19 [Dataset](https://www.kaggle.com/imdevskp/corona-virus-report?select=covid_19_clean_complete.csv)
 
-## App Sketch
+App: COVID-19 [Dashboard](https://covid-19-mds-532-group3.herokuapp.com/)
 
-The purpose of this app is to allow users to explore global COVID-19 data by time and region in an intuitive manner.
+## Welcome!
+Hello and welcome to our COVID-19 Dashboard - an interactive app that allows users to explore COVID-19 confirmed cases, deaths and recoveries around the world. 
+
+As the pandemic continues into 2021, the intent of this app is to allow users to examine the first seven months of 2020 to visualize how the pandemic progressed in the early stages. Users can view this data both on a global scale as well as on a region or country basis. The app also allows users to compare countries or regions across time periods to understand how different lockdown measures may have impacted these key measures.
+
+### App Sketch
+
+Here's an outline of the initial concept behind this app. 
 
 The landing page consists of a world heatmap which highlights regions that have been most and least impacted by the virus. Users can select the continent, country and date range they are interested in by using the respective dropdown menus and sliders. There is also a further option to specify if they are interested in absolute or per capita data through the radio buttons below. Once these fields are filled out, the bottom of the page will populate several plots with aggregated data that the user can explore in greater detail. The scroll bar will allow them to view up to six plots. 
```

Comment:
continent -> region.

we do not have sliders now, only the date range picker

Reply:
I'm not sure we need to keep updating the writing for the sketch. I suspect we will eventually remove this from the README at later milestones so I added this sentence for clarity instead:

Some design changes have been made since this initial concept and is summarized [here.](doc/reflection-milestone2.md)